src/components/BannerVideo/index.js

```diff
@@ -0,0 +1,44 @@
+import { useRef, useState, useEffect } from "react";
+import classes from "./BannerVideo.module.css";
+
+const BannerVideo = ({ data }) => {
+  const [play, setPlay] = useState(false);
+  const videoRef = useRef();
+
+  const togglePlay = () => setPlay((prev) => !prev);
+
+  useEffect(() => {
+    if (play) videoRef.current.play();
+    else videoRef.current.pause();
```

Comment:
Suggestion: you can add videoRef as in the dependency array.

Reply:
wont need it, as it wont make a difference